Dockerfile

```
@@ -1,6 +1,6 @@
 FROM python:3.10-bullseye
 
-RUN pip install "idf-component-manager~=1.2" "urllib3<2"
+RUN pip install "idf-component-manager~=1.3" "urllib3<2"
```

```suggestion
RUN pip install "idf-component-manager~=1.3"
```

There is no need to specify urllib3 version with newer component manager

---

@suda-morris I'm sorry didn't submit the comment